code/kthoom-ipfs.js

```diff
@@ -24,12 +24,17 @@ kthoom.ipfs = {
         // Load in the IPFS script API.
         const ipfsScriptEl = document.createElement('script');
         ipfsScriptEl.addEventListener('load', () => {
-          kthoom.getApp().setProgressMeter({loadPct: 0.2, label: 'Creating IPFS node...'});
-          const node = window.Ipfs.createNode();
-          node.on('start', () => {
-            kthoom.ipfs.node_ = node;
-            resolve(node);
-          });
+          if (window.ipfs) {
```

Comment:
Sadly I don't think this will work. `window.ipfs` is an instance of [js-ipfs-http-client](https://github.com/ipfs/js-ipfs-http-client), but kthoom uses a different library, [js-ipfs](https://github.com/ipfs/js-ipfs), to create an in-browser IPFS repo. This means the correct `Ipfs` object isn't present to check the prototype chain.

Reply:
Bummer.  I guess there's no properties on the object you can check?